src/main/java/seedu/toluist/controller/TaskController.java

```diff
@@ -110,4 +110,8 @@ private CommandResult delete(TodoList todoList, Task task) {
         todoList.remove(task);
         return new CommandResult(DELETE_TASK_RESULT);
     }
+
+    public static String[] getCommandWords() {
+        return new String[] { UPDATE_TASK_COMMAND, DELETE_TASK_COMMAND, ADD_TASK_COMMAND };
```

Comment:
as in i remembered one of the points prof talked about in lecture has something to do with
"Find the problem with this code", and one of the problems was something like this
UPDATE_TASK_COMMAND = "update"
ADD_TASK_COMMAND = "add"
DELETE_TASK_COMMAND = "delete"
COMMAND_WORD = "redo"
so COMMAND_WORD is out of place cuz it should be 'WORD_COMMAND'. 

tl:dr standardize the commands more can already. What you think?